kinto/core/resource/__init__.py

```diff
@@ -245,6 +245,12 @@ def collection_get(self):
         self._raise_412_if_modified()
 
         headers = self.request.response.headers
+
+        if getattr(self.context, "forced_empty_list", False):
```

Comment:
Oh I see, you have `force` and `forced`... hum, we should be able to simplify that